src/TuDa.CIMS.Web/Components/PurchaseInformation/PurchaseInformationPopup.razor.cs

```diff
@@ -0,0 +1,40 @@
+using Microsoft.AspNetCore.Components;
+using TuDa.CIMS.Shared.Dtos.Responses;
+using TuDa.CIMS.Web.Services;
+
+namespace TuDa.CIMS.Web.Components.PurchaseInformation;
+
+public partial class PurchaseInformationPopup
+{
+    private NavigationManager _navigationManager { get; set; }
+    private IPurchaseApi _purchaseApi { get; set; }
```

Comment:
🔧 To `readonly` field instead of a property
```suggestion
    private readonly NavigationManager _navigationManager;
    private readonly IPurchaseApi _purchaseApi;
```